README.rst

```diff
@@ -41,7 +41,7 @@ Supported versions
 ******************
 
 * Python 3.7, 3.8, 3.9
-* Django 2.0, 2.1, 2.2, 3.0
+* Django 2.0, 2.1, 2.2, 3.0, 3.1, 3.2
 * Django REST Framework 3.10
```

Comment:
Add also newer working versions of DRF, as far as I remember also 3.11, 3.12, not sure about 3.13 and 3.14 after drf-yasg change, probably also working.